src/app/drive/components/FileViewer/FileViewerWrapper.tsx

```diff
@@ -70,108 +63,28 @@ const FileViewerWrapper = ({
 
   const path = getAppConfig().views.find((view) => view.path === location.pathname);
   const pathId = path?.id as pathProps;
-  const isRecentsView = pathId === 'recents';
   const isSharedView = pathId === 'shared';
-  const isTrashView = pathId === 'trash';
 
-  const isSharedItem = file.sharings && file.sharings?.length > 0;
-  const isOwner = file.credentials?.user === user?.email;
-
-  const {
-    onDownloadItemButtonClicked,
-    onMoveItemButtonClicked,
-    onCopyLinkButtonClicked,
-    onShowDetailsButtonClicked,
-    onLinkSettingsButtonClicked,
-    onMoveToTrashButtonClicked,
-    onRenameItemButtonClicked,
-    onRestoreItemButtonClicked,
-    onDeletePermanentlyButtonClicked,
-  } = useDriveItemActions(currentFile);
+  const driveItemActions = useDriveItemActions(currentFile);
```

Comment:
not sure if you want to keep this global, maybe you can scope it down using a local context for each wrapper?